packages/pipeline/migrations/1542070840010-InitialSchema.ts

```diff
@@ -158,6 +158,25 @@ const transactions = new Table({
     ],
 });
 
+const token_orderbook_snapshots = new Table({
+    name: 'raw.orderbook_snapshots',
+    columns: [
+        { name: 'source', type: 'char(42)' },
+        { name: 'retrieval_timestamp', type: 'bigint' },
+        { name: 'order_type', type: 'char(42)' },
+
+        { name: 'base_asset_symbol', type: 'char(42)' },
+        { name: 'base_asset_address', type: 'char(42)' },
+        { name: 'volume_from', type: 'varchar' },
```

Comment:
`volume_from` and `volume_to` should have type `numeric`, which is a number type with arbitrary precision. This allows us to do math in queries and avoids floating point precision issues and rounding errors as much as possible.